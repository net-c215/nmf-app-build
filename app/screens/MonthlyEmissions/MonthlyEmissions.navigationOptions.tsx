import React from "react";

import { Text } from "components";
import { Layout, ComponentsStyle, Colors } from "style";

const navigationOptions = ({ route }) => ({
  ...ComponentsStyle.transitionBetweenScreenPresets,
  headerStyle: {
    ...ComponentsStyle.header,
  },
  headerBackTitleVisible: false,
  headerTintColor: Colors.grey100,
  headerTitle: () => (
    <Text.Primary style={Layout.androidNavTitle}>{route?.params?.dateHeader}</Text.Primary>
  ),
});

export default navigationOptions;
