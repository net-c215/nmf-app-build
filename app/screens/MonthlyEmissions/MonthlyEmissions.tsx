import React from "react";
import { View, Text } from "react-native";

import styles from "./MonthlyEmissions.styles";
import navigationOptions from "./MonthlyEmissions.navigationOptions";

const MonthlyEmissions = () => {
  return (
    <View style={styles.container}>
      <Text></Text>
    </View>
  );
};

MonthlyEmissions.navigationOptions = navigationOptions;

export default MonthlyEmissions;
