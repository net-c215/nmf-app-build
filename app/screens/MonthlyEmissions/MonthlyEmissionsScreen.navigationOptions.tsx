import React from "react";

import { Text } from "components";
import { Layout, ComponentsStyle, Colors } from "style";

const navigationOptions = ({ route }) => ({
  ...ComponentsStyle.transitionBetweenScreenPresets,
  headerStyle: {
    ...ComponentsStyle.header,
  },
  headerBackTitleVisible: false,
  headerTintColor: Colors.grey100,
  headerTitle: () => (
    <Text.H3 style={Layout.androidNavTitle}>{route?.params?.monthAndYear}</Text.H3>
  ),
});

export default navigationOptions;
