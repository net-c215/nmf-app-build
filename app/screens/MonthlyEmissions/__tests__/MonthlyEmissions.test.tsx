import "react-native";
import React from "react";
import { create } from "react-test-renderer";

import MonthlyEmissions from "../MonthlyEmissions";

describe("<MonthlyEmissions />", () => {
  const defaultProps = {};
  const wrapper = create(<MonthlyEmissions {...defaultProps} />);

  test("render", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
