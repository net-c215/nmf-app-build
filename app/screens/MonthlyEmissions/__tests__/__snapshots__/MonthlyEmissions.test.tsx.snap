// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MonthlyEmissions /> render 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "flex": 1,
      "paddingHorizontal": 14,
      "paddingTop": 22,
    }
  }
>
  <Text />
</View>
`;
